src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -622,15 +622,18 @@ void ResumeCtrlImpl::AddCommands(ApplicationSharedPtr application,
   if (saved_app.keyExists(strings::application_commands)) {
     const smart_objects::SmartObject& app_commands =
         saved_app[strings::application_commands];
-    for (size_t i = 0; i < app_commands.length(); ++i) {
-      const smart_objects::SmartObject& command = app_commands[i];
+
+    uint32_t internal_id = 1;
```

Comment:
@APipko magic number? At least leave a comment why it is 1. In such cases it is better to use const named variables.